File: Ex17.c
#include<stdio.h>



int main(int argc, char const *argv[])
{
    // Inserção das variáveis

    int NumeroX, NumeroY, NumeroZ, Hipo;

    // Inserção do primeiro valor
    printf("Insira um valor qualquer: ");
    scanf("%i",&NumeroX);

    
    // Inserção do segundo valor
    printf("Insira um valor qualquer: ");
    scanf("%i",&NumeroY);
    
    
    // Inserção do terceiro valor
    printf("Insira um valor qualquer: ");
    scanf("%i",&NumeroZ);

    // Comparação para saber o maior número
    if (NumeroX>NumeroY)
    {
        if (NumeroX>NumeroZ)
        {
            Hipo= NumeroX*NumeroX;
            if (Hipo==(NumeroY*NumeroY)+(NumeroZ*NumeroZ))
            {
                printf("Com estes valores obteremos um triangulo retangulo.");
            }
            else
            {
                printf("Com estes valores nao obteremos um triangulo retangulo.");
            }
            
        }
        else
        {
            Hipo= NumeroZ*NumeroZ;
            if (Hipo==(NumeroY*NumeroY)+(NumeroX*NumeroX))
            {
                printf("Com estes valores obteremos um triangulo retangulo");
            }
            else
            {
               printf("Com estes valores nao obteremos um triangulo retangulo."); 
            }
        }
        
    }
    

    return 0;
}

File: Ex17.py
a = float(input("Insira um numero qualquer: "))
b = float(input("Insira um outro numero qualquer: "))
c = float(input("Insira um terceiro numero qualquer: "))

if a > b and a > c:

    hipo = a*a
    if hipo == ((b*b)+(c*c)):
        print("Os valores inseridos formam um triangulo retangulo")

    else:
        print("Os valores inseridos nao formam um triangulo retangulo") 

if b > a and b > c:
        hipo = b*b

        if hipo == ((a*a)+(c*c)):
            print("Os valores inseridos formam um triangulo retangulo")

        else:
             print("Os valores inseridos nao formam um triangulo retangulo")

if c > a and c > b:
        hipo = c*c

        if hipo == ((a*a)+(b*b)):
            print("Os valores inseridos formam um triangulo retangulo")

        else:
             print("Os valores inseridos nao formam um triangulo retangulo")
             